docs/roadmap.markdown

```diff
@@ -4,9 +4,122 @@ title: "Roadmap"
+| New Relic configuration can be inconsistent across applications | <ul><li>Manual configuration is required<li>Auditing configurations is labor intensive</ul> | Provide tools so New Relic configurations can be templated, repeatable, and audited easily |
+| I cannot integrate New Relic into my exiting workflow | <ul><li>Using New Relic in a workflow currently requires significant time investment in custom scripts</ul> | Provide easy to consume tooling that can be leveraged in existing workflows to streamline New Relic usage |
+
+
+## Personas
+
+**Software Engineer** - A software engineer is someone who is writing, deploying, and managing the lifecycle of any software stack that utilizes New Relic. As the owner of the application, they are responsible for managing all Alerts, Dashboards, Nerdpacks, and other New Relic resources specific to their application.
+
+**Site Reliability Engineer** - A Site Reliability Engineer (SRE) is an individual who is responsible for the stability of the entire software stack.  They assist Software Engineers with proper New Relic configuration, and provide overall guidance across teams.  Working with multiple teams becomes difficult when they do not operate in a similar way, so SREs strive to develop easy to follow standards.
+
+**Security Engineer** - A Security Engineer is someone responsible for auditing and verify compliance of their software.  In order to facilitate this, New Relic has been made a requirement for all services, and as such needs specific tagging and configuration across all assets.  When faced with any reasonable sized organization this becomes a daunting task.
+
+
+## Job Stories
+
+**Observability As Code Providers**
+
+* When I deploy a service, I want to embed the New Relic configuration in code with the rest of the service definition
+* When I audit our systems, I want an easy way to verify the New Relic configuration is defined as expected
+
+**New Relic CLI**
+
+* When I deploy an applications, I want to record the deployment in New Relic without having to learn how their API works
+* When service ownership changes, I want an easily, automated way to change tags within New Relic
+
+**New Relic Client**
+
+* When I have custom tooling in place that I want to leverage New Relic, I want to be able to use a supported library.
+* When I build a deployment pipeline using serverless functions, I want to include New Relic.
+
+
+
+## Developer Toolkit v1
+
+To solve for the above problems, the initial version of the Developer Toolkit is defined to contain the following resources.  All resources shall be built as Open Source projects, and actively engage with the broader community.
```

Comment:
I would just say "the initial version of the Developer Toolkit will contain the following resources" vs defined to contain.